packages/date/src/dateRangeElement.ts

```diff
@@ -0,0 +1,68 @@
+import { html, LitElement, property, TemplateResult } from 'lit-element';
+import { formatDistance, formatISO, formatRelative, Locale } from 'date-fns';
+import { enGB } from 'date-fns/locale';
+import parseISO from 'date-fns/parseISO';
+
+export type DateRangeVariant = 'distance' | 'relative';
+
+export type WeekDay = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface DateRangeElementProps {
+  addSuffix?: boolean;
+  baseDate?: string;
+  date: string;
+  includeSeconds?: boolean;
+  locale?: Locale;
+  variant?: DateRangeVariant;
+  weekStartsOn?: WeekDay;
+}
+
+export class DateRangeElement extends LitElement implements DateRangeElementProps {
+  @property({ type: Boolean })
+  addSuffix?: boolean = false;
```

Comment:
preferably do not use camelcase (event tho browser are case insensitive, all other elements only use lower cap or snakecase), rename to `sufix`